SecretKit/Common/Erasers/AnySecretStore.swift

```diff
@@ -49,10 +49,12 @@ public class AnySecretStoreModifiable: AnySecretStore, SecretStoreModifiable {
 
     private let _create: (String, Bool) throws -> Void
     private let _delete: (AnySecret) throws -> Void
+    private let _updateName: (AnySecret, String) throws -> Void
```

Comment:
Minor nit: please keep the `_` erased closure matching the public function name (just with an underscore prefix)
```suggestion
    private let _update: (AnySecret, String) throws -> Void
```